### PR TITLE
fix(candeletecardinalities): return correct response on route negative case (DEV-36)

### PR DESCRIPTION
**Pull request by mpro7**
resolves DEV-36

### CODE REVIEW

**Review by irinaschubert — APPROVED** [review]

File: webapi/src/test/scala/org/knora/webapi/e2e/v2/OntologyV2R2RSpec.scala [hunk 1]
```diff
@@ -78,7 +78,20 @@ class OntologyV2R2RSpec extends R2RSpec {
   private val clientTestDataPath: Seq[String] = Seq("v2", "ontologies")
 
   // Collects client test data
-  private val clientTestDataCollector = new ClientTestDataCollector(settings)
+  // TODO: redefine below method somewhere else if can be reused over other test files
+  private def CollectClientTestData(filename: String, payload: String) = {
```
Comment (irinaschubert) [inline webapi/src/test/scala/org/knora/webapi/e2e/v2/OntologyV2R2RSpec.scala]:
```suggestion
  private def collectClientTestData(filename: String, filecontent: String) = {
```

**Review by irinaschubert — COMMENTED** [review]
I like the removal of duplicated code. Did you test if it is working? Because there is now a new `clientTestDataCollector` created in each call of `CollectClientTestData(...)`. Before, it was one `clientTestDataCollector` for all the tests in this file. Do we still get all test data in the end?

File: webapi/src/test/scala/org/knora/webapi/e2e/v2/OntologyV2R2RSpec.scala (hunk 1 above)
Comment (mpro7) [reply webapi/src/test/scala/org/knora/webapi/e2e/v2/OntologyV2R2RSpec.scala]: @irinaschubert  at the end it's both payload and the file content ;) but in the matter of consistency I accept this suggestion.

`val clientTestDataCollector = new ClientTestDataCollector(settings)` can be also defined outside the method, maybe this is better performance wise. Everything was working well, but I think that some tests failed after that commit 2d9c26d where we added 3 missing test data.

**Review by subotic — APPROVED** [review]
LGTM, thanks!

**Pull request merged**